apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -0,0 +1,195 @@
+    const WillNotReceiveLabelMessage = intl.formatMessage({ id: 'news.component.RecipientCounter.willNotReceive.label' })
+    const WillNotReceiveHintMessage = intl.formatMessage({ id: 'news.component.RecipientCounter.willNotReceive.hint' })
+    const formatWillReceiveHintMessage = (count) => intl.formatMessage({ id: 'news.component.RecipientCounter.willReceive.hint' }, { count })
+
+    const { organization } = useOrganization()
+
+    const propertyIdsFromNewsItemScopes = compact(uniq(map(newsItemScopes, 'property.id')))
+
+    const { objs: properties, loading: loadingProperties } = Property.useAllObjects({
+        where: {
+            organization: { id: organization.id },
+            ...(propertyIdsFromNewsItemScopes.length > 1 ? { id_in: propertyIdsFromNewsItemScopes } : {}),
+            deletedAt: null,
+        },
+    })
+
+    const { objs: residents, loading: loadingResidents } = Resident.useAllObjects({
+        where: queryFindResidentsByNewsItemAndScopes(organization.id, newsItemScopes),
+    })
+
+    if (loadingProperties || loadingResidents) {
+        return null
+    }
+
+    const message = buildMessageFromNewsItemScopes(newsItemScopes, intl)
+
+    // NOTE(antonal): Not all corner cases are handled, because they rarely will occur:
+    // - When some records of NewsItemScope are connected to Property and some are not
+    // - When some records of NewsItemScope have unitName, that does not exist in connected Property
+    // - When some Resident's unitName is out of units range of a property, that can happen if Property.map was changed after Resident was registered
+
+    const propertiesWillReceive = newsItemScopes.length === 0 || newsItemScopes.length === 1 && isTargetedToOrganization(newsItemScopes[0])
+        ? properties
+        : properties.filter(p => propertyIdsFromNewsItemScopes.includes(p.id)) // Take Property objects with full set of fields, obtained using Property client utils
+
+    const unitsWillReceive = uniqBy(residents, ({ property, unitName }) => [property.id, unitName].join('-'))
+
+    const willNotReceiveUnitsCount = calculateWillNotReceiveCount(residents, propertiesWillReceive)
+
+    return (
+        <div style={{ maxWidth: '500px' }}>
+            <Card>
+                <Space direction='vertical' size={24} width='100%'>
+                    <Typography.Text>{MailingMessage} <Typography.Text strong>{message}</Typography.Text></Typography.Text>
+                    <JustifiedSpace direction='horizontal' align='start' size={24} width='100%'>
+                        <Counter label={PropertiesLabelMessage} value={propertiesWillReceive.length}/>
+                        <Counter label={WillReceiveLabelMessage} value={unitsWillReceive.length} hint={formatWillReceiveHintMessage(unitsWillReceive.length)}/>
+                        <Counter label={WillNotReceiveLabelMessage} value={willNotReceiveUnitsCount} hint={WillNotReceiveHintMessage}/>
+                    </JustifiedSpace>
```

Comment:
Please use something like the following instead of `emotion`-styled components
```suggestion
                    <Col xs={24}>
                        <Row align='top' justify='space-evenly'>
                            <Counter label={PropertiesLabelMessage} value={propertiesWillReceive.length}/>
                            <Counter label={WillReceiveLabelMessage} value={unitsWillReceive.length} hint={formatWillReceiveHintMessage(unitsWillReceive.length)}/>
                            <Counter label={WillNotReceiveLabelMessage} value={willNotReceiveUnitsCount} hint={WillNotReceiveHintMessage}/>
                        </Row>
                    </Col>
```